test/extension.test.ts

```diff
@@ -68,8 +70,7 @@ suite("Arduino: Extension Tests", () => {
         // When running test on osx, the vscode instance is hanging there after tests finished and cause mocha timeout.
```

Comment:
We can imrpove the comment here. This action of stoping usb detector listening works for botj osx and windows test.

Reply:
updated.